contexts/ed25519-signature-2020-v1.json

```diff
@@ -0,0 +1,74 @@
+{
+  "@context": {
+    "@version": 1.1,
```

Comment:
We should really stop proliferating `@version` everywhere, it is unnecessary. We should also confirm that jsonld.js does not consider the `@version` field when comparing protected term definitions. It shouldn't do that -- and if it doesn't, there's definitely no reason to include `@version` here or anywhere.

Reply:
Oh cool, good to know. Will remove.